resend.yaml

```diff
@@ -2127,499 +3215,580 @@ components:
       type: object
       properties:
         id:
           type: string
-          description: The ID of the duplicated template.
+          description: The ID of the contact.
+          example: 'e169aa45-1ecf-4183-9955-b1499d5701d3'
         object:
           type: string
-          description: The object type of the response.
-          example: template
+          description: Type of the response object.
+          example: contact
+        deleted:
+          type: boolean
+          description: Indicates whether the contact was successfully removed from the segment.
+          example: true
+    ListContactSegmentsResponseSuccess:
+      type: object
+      properties:
+        object:
+          type: string
+          description: Type of the response object.
+          example: list
+        has_more:
+          type: boolean
+          description: Indicates if there are more segments to retrieve.
+          example: false
+        data:
+          type: array
+          description: Array containing segment information.
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The ID of the segment.
+                example: '78261eea-8f8b-4381-83c6-79fa7120f1cf'
+              name:
+                type: string
+                description: The name of the segment.
+                example: 'Registered Users'
+              created_at:
+                type: string
+                format: date-time
+                description: Timestamp indicating when the segment was created.
+                example: '2023-10-06T22:59:55.977Z'
+    ListContactTopicsResponseSuccess:
+      type: object
+      properties:
+        object:
+          type: string
+          description: Type of the response object.
+          example: list
+        has_more:
+          type: boolean
+          description: Indicates if there are more topics to retrieve.
+          example: false
+        data:
+          type: array
+          description: Array containing topic subscription information.
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The ID of the topic.
+                example: 'b6d24b8e-af0b-4c3c-be0c-359bbd97381e'
+              name:
+                type: string
+                description: The name of the topic.
+                example: 'Weekly Newsletter'
+              subscription:
+                type: string
+                enum:
+                  - opt_in
+                  - opt_out
+                description: The subscription status.
+                example: 'opt_in'
+              created_at:
+                type: string
+                format: date-time
+                description: Timestamp indicating when the topic was created.
+                example: '2023-04-08T00:11:13.110779+00:00'
+    UpdateContactTopicsOptions:
```

Comment:
Something feels weird here:

The docs say topics should be a parameter in the body: https://resend.com/docs/api-reference/contacts/update-contact-topics

But the API seems to validate them as being in the root of the request?

https://github.com/resend/resend-api/blob/1467ffa9af287dd254dcfd8d9b5f9c60cc4d710c/src/routes/contacts/topics/validations/topic-subscriptions-schema.ts#L11-L16

https://github.com/resend/resend-api/blob/1467ffa9af287dd254dcfd8d9b5f9c60cc4d710c/src/routes/contacts/topics/validations/patch.ts#L17